code/modules/mob/mob.dm

```diff
@@ -1103,12 +1103,18 @@
 
 /mob/living/proc/handle_stunned()
 	if(stunned)
-		AdjustStunned(-1)
+		if(mind?.changeling)
+			AdjustStunned(-2)
```

Comment:
feel like this stuff should be on /human/ instead?

Also there's a pretty big chance that you'll need to do some more shit with brute/burn mod in the future, so it may be worth to make a mutable species data datum on the human, dump all mutable vars on there like burn mod, brute mod, etc and use those here, but eh, don't need to for now i guess?

Reply:
It's on living so lesser form can be affected by it